tests/testthat/helper-return-cleaners.R

```diff
@@ -0,0 +1,13 @@
+# This fn removes calls from function returns to make testing easier
+rmcall <- function(obj) {
+  if (!is.null(obj[["call"]])) {
+    obj[["call"]] <- NULL
+  }
+  return(obj)
```

Comment:
```
rmcall <- function(obj) { structure(obj, call=NULL) }
```

Reply:
ah - right, different from DeclareDesign. You can still remove the null check I think.